api/swagger-spec/v1.json

```diff
@@ -13262,6 +13261,10 @@
      "revision": {
       "type": "string",
       "description": "Commit hash for the specified revision."
+     },
+     "directory": {
+      "type": "string",
+      "description": "Target directory name"
```

Comment:
I think the behavior of this option should be fully spelled out here.  How about:

```
Target directory name.  Must not contain '..' or spaces.  If '.' is supplied, the volume directory will be the git repository.  Otherwise, if specified, the volume will contain the git repository in the subdirectory with the given name.
```